cookiecutter-django-ida/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py

```diff
@@ -26,15 +23,12 @@ def get_logger_config(log_dir='/var/tmp',
     """
 
     hostname = platform.node().split(".")[0]
-    syslog_format = (
-        "[service_variant={service_variant}]"
-        "[%(name)s][env:{logging_env}] %(levelname)s "
-        "[{hostname}  %(process)d] [%(filename)s:%(lineno)d] "
-        "- %(message)s"
-    ).format(
-        service_variant=service_variant,
-        logging_env=logging_env, hostname=hostname
-    )
+    syslog_format = ("[service_variant={service_variant}]"
+                     "[%(name)s][env:{logging_env}] %(levelname)s "
+                     "[{hostname}  %(process)d] [user %(userid)s] [ip %(remoteip)s] [%(filename)s:%(lineno)d] "
+                     "- %(message)s").format(service_variant=service_variant,
+                                             logging_env=logging_env,
+                                             hostname=hostname)
```

Comment:
Would you be amenable to preserving the original indentation? That way this section would just be a one-line diff, easier to review here and to read in the git history.